services/QuillLMS/client/app/bundles/Teacher/containers/dashboard.jsx

```diff
@@ -41,11 +41,10 @@ const Dashboard = ({ onboardingChecklist, firstName, mustSeeModal, linkedToCleve
     getMetrics();
     getDiagnostics()
     getLessons()
-    getActivityFeed()
   }, []);
 
   React.useEffect(() => {
-    if (metrics && diagnostics && lessons && activityFeed && loading) {
+    if (metrics && diagnostics && lessons && loading) {
       setLoading(false)
     }
   }, [metrics, diagnostics, lessons, activityFeed])
```

Comment:
Do you need to remove this `activityFeed`?. Also possibly on line 38?

Reply:
Nope! Here it's saying "call this function if the value of `activityFeed` (or `metrics` or `diagnostics` or `lessons`) changes", so it'll work fine for those other ones but never get called because the value of activityFeed will never change from how it was originally defined in line 37 (as null). If we were removing this permanently I would take this out, but leaving it in isn't doing anything right now.